### PR TITLE
Properly test packages, add galaxy-test-tools and galaxy-model-tools package

### PR DESCRIPTION
**Pull request by mvdbeek**
Builds on https://github.com/galaxyproject/galaxy/pull/8358

### CODE REVIEW

File: packages/app/setup.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def get_var(var_name):
     'galaxy.tools.error_reports',
     'galaxy.tools.expressions',
     'galaxy.tools.filters',
-    'galaxy.tools.imp_exp',
+    'galaxy.model_tools.imp_exp',
```
Comment (nsoranzo) [inline packages/app/setup.py]:
```suggestion

```

and add `galaxy-model-tools` to `packages/app/requirements.txt`

**Comment by jmchilton** [comment]
Fascinating... wasn't a direction I was expecting but looks good.

**Comment by jxtx** [comment]
Cool, but why "model_tools". The name is not immediately intuitive to me.

**Comment by mvdbeek** [comment]
No good reason, we've been calling them that way in the codebase, they're also known as Database Operation tools. We could also call them "special tools", "framework tools" or something else altogether. Do you have a suggestion ?

**Comment by jmchilton** [comment]
This will break import/export jobs that occur during an upgrade I think. We broke those last cycle and no one noticed. The little we tried to do to save them just turned out to be unneeded complexity and wasted effort IMO - so I'd just like to do that one more time and keep this as is without worrying about these potential jobs.

**Comment by mvdbeek** [comment]
I think we may be able to rescue those jobs if we provide a fallback import at the old location -- that should be possible, right ? `In tools/__init__.py`
```
from galaxy.model_tools import imp_exp  # noqa: whatever the right number is  # fallback for old imp/exp jobs
```

**Comment by jmchilton** [comment]
I mean - we could definitely fix them. I don't think it is worth adding the lines of code to do that though. It will likely not really save a job across all Galaxies everywhere IMO.

**Comment by jxtx** [comment]
Framework tools makes more sense to me.
-- 
jt (mobile)

**Comment by jmchilton** [comment]
😿 this was cool stuff - hopefully we find some time to keep it going at some point.

**Pull request closed without merging**